tmt-app/src/Components/MajorCareerList.jsx

```diff
@@ -25,7 +26,28 @@ function MajorCareerList() {
                 padding: "0.25rem 0",
               }}
             >
-              {item.name}
+              <Link
+                to={`/majors/${item.key}`}
+                component={RouterLink}
+                sx={{
+                  color: "Boba.main",
+                  textDecoration: "underline",
```

Comment:
Is there a default styling for this? Do we need to set underline?

Reply:
I believe the default will look a bit different cause of the color and size

Reply:
![image](https://user-images.githubusercontent.com/36683390/235341655-82795bd4-eb8f-42bd-9ff3-f5bd2af68994.png)
This is the default without setting underline